File: PracticallClass/DJANGO/notes/flutter/noteapp/lib/screens/urls.dart
// urls.dart
class ApiUrls {
  // Základní URL pro Django API
  static const String baseUrl = 'http://127.0.0.1:8000/';

  // Endpointy pro poznámky
  static String getNotes() {
    return '${baseUrl}notes/';
  }

  static String getNoteById(int id) {
    return '${baseUrl}notes/$id/';
  }

  static String createNote() {
    return '${baseUrl}notes/create/';
  }

  static String updateNote(int id) {
    return '${baseUrl}notes/update/$id/';
  }

  static String deleteNote(int id) {
    return '${baseUrl}notes/delete/$id/';
  }
}

File: PracticallClass/DJANGO/notes/api/urls.py
from django.urls import path
from . import views

urlpatterns = [
    path('', views.getRoutes),
    path('notes/', views.getNotes),  # pointer na get notes
    path('notes/<int:pk>/', views.getNoteById),
    path('notes/create/', views.createNote),
    path('notes/update/<int:pk>/', views.updateNote),
    path('notes/delete/<int:pk>/', views.deleteNote),
]